hera_pspec/uvpspec.py

```diff
@@ -1721,17 +1718,15 @@ def get_exact_window_functions(self, ftbeam_file=None, spw_array=None,
                 # initialise UVWindow object
                 uvw = UVWindow.from_uvpspec(self, ipol=i, spw=spw, ftfile=ftbeam_file,
                                             x_orientation=x_orientation, verbose=verbose)
-                
                 # extract kperp bins the window functions corresponding to the baseline 
                 # lengths given as input
                 kperp_bins = uvw.get_kperp_bins(blpair_lens)
                 kpara_bins = uvw.get_kpara_bins(uvw.freq_array)
                 pol_window_function = np.zeros((self.Nblpairts, self.get_dlys(spw).size, kperp_bins.size, kpara_bins.size))
-
                 # Iterate over baseline-pair groups
                 for j, blpg in enumerate(blpair_groups):
                     if verbose: 
-                        sys.stdout.write('\rComputing for bl group {} of {}...'.format(j+1,len(blpair_groups)))
+                        sys.stdout.write('\rComputing for bl group {} of {}...'.format(j+1, len(blpair_groups)))
```

Comment:
why is this not just a print statement, or `logging` statement?

Reply:
Changed to print